README.md

```diff
@@ -92,33 +92,35 @@ Change directory into the freshly cloned repo.
 Once you have cloned and changed directories into the freshly cloned repository
 you have three methods to choose from:
 
-#### Method 1: Using Docker (Safest)*
+#### Method 1: Using Podman (Safest)*
 
 [![Docker Image Version (tag latest semver)](https://img.shields.io/docker/v/qasimwarraich/cli-tutor/latest?label=docker)](https://hub.docker.com/r/qasimwarraich/cli-tutor)
 
 Prerequisites:
 
 - go v1.18 or higher
-- docker
+- [Podman](https://podman.io/)
 - make (optional)
 
 Instructions:
 
 Option 1: Pull down the image from [Docker Hub](https://hub.docker.com/r/qasimwarraich/cli-tutor)
 
-`docker pull qasimwarraich/cli-tutor`
+`podman pull qasimwarraich/cli-tutor`
```

Comment:
Docker hub is not a default source in `Podman`. Furthermore we should point to the `ghcr` image.

Reply:
Yea, I had to set dockerhub in the config file as a source. Didn't work otherwise. Is this the valid `ghcr` link?
```code
docker pull ghcr.io/qasimwarraich/cli-tutor:latest
```
I'd then change `docker` into `podman and set it.

Reply:
Changed the container source.  Not sure if we should add the changes to the `Makefile` here? Probably not.

Reply:
That link is correct.